src/hclib-runtime.c

```diff
@@ -1096,9 +1129,8 @@ void help_finish(finish_t *finish) {
 
 void hclib_start_finish() {
     hclib_worker_state *ws = CURRENT_WS_INTERNAL;
-    finish_t *finish = (finish_t *) HC_MALLOC(sizeof(finish_t));
+    finish_t *finish = malloc(sizeof(*finish));
```

Comment:
I would be happy with this, I'm not sure what the original motivation of them was.